db.js

```diff
@@ -8,4 +8,12 @@ admin.initializeApp({
 });
 
 const db = admin.database();
+
+export const getUser = async (message) => {
```

Comment:
typically, a function like this takes in an `id` and returns the object -- so you'll want to be given the `userId` as an argument instead of a "message"